packages/tests/server/interop/regression/issue-949-inferProcedureInput.test.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Any particular reason for this one? I tried switching the `{}`s to `object`s in it and didn't see any build failures.